stable/yugaware/templates/statefulset.yaml

```diff
@@ -82,17 +92,33 @@ spec:
         - image: {{ include "full_yugaware_image" . }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           name: prometheus-configuration
-          command: ["cp", "/default_prometheus_config/prometheus.yml", "/prometheus_configs/prometheus.yml"]
           {{- if .Values.securityContext.enabled }}
+          command:
+            - 'bash'
+            - '-c'
+            - >
+              cp /default_prometheus_config/prometheus.yml /prometheus_configs/prometheus.yml;
+              bash /init-container/init-permissions.sh;
```

Comment:
if this script fails, the error may not be propagated to the bash -c launcher unless we either set -e on this small piece OR run it as `cp ... && bash init-permissions.sh`